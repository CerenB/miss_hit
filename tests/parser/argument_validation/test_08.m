% From https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html

c = test_07;
d = test_07(2);
e = test_09(2, 3);
