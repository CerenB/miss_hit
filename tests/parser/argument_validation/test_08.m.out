----------------------------------------------------------------------
--  Parse tree for test_08.m
Root: Script_File
  Statements: Sequence_Of_Statements
    Statements: Simple_Assignment_Statement
      LHS: Identifier <c>
      RHS: Identifier <test_07>
    Statements: Simple_Assignment_Statement
      LHS: Identifier <d>
      RHS: Reference
        Name: Identifier <test_07>
        Arguments: Number_Literal <2>
    Statements: Simple_Assignment_Statement
      LHS: Identifier <e>
      RHS: Reference
        Name: Identifier <test_09>
        Arguments: Number_Literal <2>
        Arguments: Number_Literal <3>
----------------------------------------------------------------------
In test_08.m, line 3
| c = test_07;
|            ^ info: this token is not linked to the ast
In test_08.m, line 4
| d = test_07(2);
|            ^ info: this token is not linked to the ast
In test_08.m, line 4
| d = test_07(2);
|              ^ info: this token is not linked to the ast
In test_08.m, line 4
| d = test_07(2);
|               ^ info: this token is not linked to the ast
In test_08.m, line 5
| e = test_09(2, 3);
|            ^ info: this token is not linked to the ast
In test_08.m, line 5
| e = test_09(2, 3);
|              ^ info: this token is not linked to the ast
In test_08.m, line 5
| e = test_09(2, 3);
|                 ^ info: this token is not linked to the ast
In test_08.m, line 5
| e = test_09(2, 3);
|                  ^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
